packages/web/src/pages/sign-up-page/components/AccountHeader.tsx

```diff
@@ -84,7 +84,7 @@ export const AccountHeader = (props: AccountHeaderProps) => {
     onCoverPhotoChange,
     size
   } = props
-  const { value: profileImage } = useSelector(getProfileImageField) ?? {}
+  const profileImage = useSelector(getProfileImageField)
```

Comment:
this makes sense, but what about the selectors for legacy sign-up?

Reply:
i ran a typecheck and updated all of the places in web that use this. I'll double check that signing up with the old flow still works though 👍

Reply:
confirmed, this works with the old flow as well